recipes/jupyter-book/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "jupyter-book" %}
+{% set version = "0.8.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 77da2c5714494d0a1eed494542f1a717a7addbb0dfa30b1f7292f7a4caf0fb0c
+
+build:
+  number: 0
+  entry_points:
+    - jb = jupyter_book.commands:main
+    - jupyter-book = jupyter_book.commands:main
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools
+  run:
+    - click
+    - docutils >=0.15
+    - jsonschema
+    - myst-nb >=0.10.1,<0.11
+    - nbconvert <6
+    - nbformat
```

```suggestion
    - nbformat <6
```

I think its best to pin as much as possible now to an upper limit, because if you pin in a later version Conda can end up giving people an earlier version, to avoid incompatibilities